src/pages/TeamPage/index.js

```diff
@@ -0,0 +1,349 @@
+                >
+                  Volunteer at Meraki
+                </Button>
+              </Grid>
+            </Stack>
+          </Container>
+          <Container
+            className={
+              !isActive
+                ? classes.team_containerTopSpace
+                : `${classes.team_responsiveContainer}`
+            }
+          >
+            <Grid container style={{ justifyContent: "center" }}>
+              <Typography
+                onClick={() => {
+                  setMembers({ volunteers: false, teamMembers: true });
+                }}
+                variant="subtitle1"
+                className={
+                  !isActive
+                    ? classes.team_selector
+                    : classes.team_MobileSelector
+                }
+                style={
+                  members.teamMembers
+                    ? {
+                        fontWeight: "bold",
+                        borderBottom: "3px solid #48a145",
+                      }
+                    : {}
+                }
+              >
+                Core Team
+              </Typography>
+              <Typography
+                onClick={() => {
+                  setMembers({ teamMembers: false, volunteers: true });
+                }}
+                variant="subtitle1"
+                className={
+                  !isActive
+                    ? classes.team_selector
+                    : classes.team_MobileSelector
+                }
+                style={
+                  members.volunteers
+                    ? {
+                        fontWeight: "bold",
+                        borderBottom: "3px solid #48a145",
+                      }
+                    : {}
+                }
+              >
+                Our Supporters
+              </Typography>
+            </Grid>
+            <Container
+              className={
+                !isActive
+                  ? `${classes.team_infoCardContaier} team-info-cards-container`
+                  : `${classes.team_infoResponsiveContainer} team-info-cards-container`
+              }
+              sx={{ marginTop: 4 }}
+            >
+              <Grid container>
+                {Object.keys(team).length ? (
+                  shuffleArray(Object.keys(team)).map((item) => {
+                    const condition = members.teamMembers
+                      ? "teamMembers"
+                      : "volunteers";
+                    if (
+                      (condition === "volunteers" &&
+                        team[item].Association === "Volunteer") ||
+                      (condition === "teamMembers" &&
+                        team[item].Association !== "Volunteer")
+                    ) {
+                      if (
+                        team[item].Photo &&
+                        team[item].Name &&
+                        team[item].Content.length &&
+                        team[item].Content &&
```

Comment:
If `team[item].Content` is `undefined` or `null`, there would be an error if you wrote it the other way around. 
```suggestion
                        team[item].Content &&
                        team[item].Content.length &&
```